src/algebra/group_with_zero/basic.lean

```diff
@@ -843,6 +843,14 @@ classical.by_cases
   (assume ha, ha)
   (assume ha, ((one_div_ne_zero ha) h).elim)
 
+lemma range_mul_left_eq_univ {a : G₀} (h : a ≠ 0) :
+  set.range (λ g, a * g) = set.univ :=
+set.eq_univ_iff_forall.mpr $ λ g, ⟨a⁻¹ * g, by simp [← mul_assoc, mul_inv_cancel h]⟩
```

Comment:
```suggestion
mul_left_surjective₀.range_eq
```
where `mul_left_surjective₀` should be be the `group_with_zero` version of `mul_left_surjective`.